### DISCUSSION TITLE
chr22_KI270879v1_alt and Depth of Coverage Files

### DISCUSSION THREAD

**Discussion by emeadd** [pypgx/pypgx#90] [Q&A] [answered]
Hello.  I tried to generate a depth of coverage file using GRCh38 and got the following message: 
SamtoolsError: 'samtools returned with error 1: stdout=, stderr=samtools depth: cannot parse region "chr22_KI270879v1_alt:267307-281486"\n'
How can I resolve this issue?  Thank you in advance for your help.

**Comment by sbslee** [comment, marked as answer]
Hello @emeadd,

Please see #65 and the [FAQ](https://pypgx.readthedocs.io/en/latest/faq.html#chr22-ki270879v1-alt-in-grch38). Please let me know if you need further help.
